src/Middleware/UserPageDecorationMiddleware.php

```diff
@@ -79,14 +80,15 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         }
         if (method_exists($page, 'getFeedbackPanel')
             && $user->hasPermission('bvl_feedback')
+            && isset($get['candID'])
         ) {
             $tpl_data['feedback_panel'] = $page->getFeedbackPanel(
-                $get['candID'],
+                $candID,
                 $get['sessionID'] ?? null
             );
 
-            $tpl_data['bvl_feedback'] = NDB_BVL_Feedback::bvlFeedbackPossible(
-                $this->page
+            $tpl_data['bvl_feedback'] = \NDB_BVL_Feedback::bvlFeedbackPossible(
+                $page
```

Comment:
should be `$this->PageName`
(will solve `1) InstrumentListTestIntegrationTest::testPageUIs`)